### DISCUSSION TITLE
Design Improvements (Fluerial Theme) and Suggestions for Enhancing the Floorp Browser

### DISCUSSION THREAD

**Discussion by midral** [Floorp-Projects/Floorp#614] [Feature Request]
![FLOORP_DESIGN_IMPROVEMENTS_AND_SUGGESTIONS_MIDRAL_03_12_23_S1](https://github.com/Floorp-Projects/Floorp/assets/152922509/d8626c26-7c45-49d6-9fd2-1673519c5a7d)

![FLOORP_DESIGN_IMPROVEMENTS_AND_SUGGESTIONS_MIDRAL_03_12_23_S2](https://github.com/Floorp-Projects/Floorp/assets/152922509/a932f147-0cc8-4315-a4d1-a949b81a1f75)

![FLOORP_DESIGN_IMPROVEMENTS_AND_SUGGESTIONS_MIDRAL_03_12_23_S3](https://github.com/Floorp-Projects/Floorp/assets/152922509/2a4c6f3a-4647-469a-b527-834474029058)

![FLOORP_DESIGN_IMPROVEMENTS_AND_SUGGESTIONS_MIDRAL_03_12_23_S4](https://github.com/Floorp-Projects/Floorp/assets/152922509/21418e19-686a-4f48-b977-e384169e3b95)

Here the document as PDF:
[FLOORP_DESIGN_IMPROVEMENTS_AND_SUGGESTIONS_MIDRAL_03_12_23.pdf](https://github.com/Floorp-Projects/Floorp/files/13551159/FLOORP_DESIGN_IMPROVEMENTS_AND_SUGGESTIONS_MIDRAL_03_12_23.pdf)

**Comment by CutterKnife** [comment]
First, I would like to express my deepest appreciation for your specific suggestions. I am not an expert in UI design, but I can agree with you on much of what it says.
Currently, the Floorp team is significantly under-resourced in development, so they are not able to maintain all of the functionality. (In fact, some themes have been "deprecated".)
The Fluerial theme is maintained not by Surapunoyousei, the leader of the Floorp project, but by the same developer as the leader of TUIC. However, he does not seem to be able to spend much time on it either.
So it is unlikely that Surapunoyousei will work on improving the theme in the near future unless a contributor shows up. Really sorry.
(In terms of functionality, a tab stacking feature is under development to replace Workspaces. It is expected to be more stable.)
Once again, thank you for making a serious effort to improve Floorp.

*This text has been translated from Japanese by DeepL. Sorry if there are any errors.

**Reply by midral** [reply]
Thank you for your kind and prompt response, as well as for the information you have just provided. I think Floorp development is on a promising path; however, it will take some time to implement and revise all the features. As there will be a growing number of Fluerial UI users, it might be wise to continue improvements for this UI. 
Again, thank you for your reply.
